mtags/src/main/scala-3/scala/meta/internal/pc/CompletionValue.scala

```diff
@@ -0,0 +1,24 @@
+package scala.meta.internal.pc
+
+import dotty.tools.dotc.interactive.Completion
+
+enum CompletionValue {
```

Comment:
Yes, it's kind of ScopeMember but not exact. Added it for better completion ordering.
We will have to rework it because currently it just marks the source of completion: if it comes from Compiler or from WorkspaceSearch etc.